.github/ISSUE_TEMPLATE/release_card_template

```diff
@@ -0,0 +1,33 @@
+---
+name: Release card template
+about: Template to use when build cards are done and we're ready to release a new or updated component / template 
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Summary:** Release of the XXXX component. 
```

Comment:
Also, we can add something here that says "Use the title format `Publish: ` for this type of ticket"